scripts/qa_crowbarsetup.sh

```diff
@@ -2735,7 +2735,7 @@ function onadmin_addupdaterepo()
 {
     pre_hook $FUNCNAME
 
-    local UPR=$tftpboot_repos_dir/Cloud-PTF
```

Comment:
This would break testing of older versions. You could do
iscloudver 6plus && UPR=.../PTF